cypress/integration/smokeTest.spec.js

```diff
@@ -18,4 +20,11 @@ describe('Smoke test', () => {
     assertFrameContains('Foo');
     assertCodePaneContains('<Foo color="blue" />');
   });
+
+  it('formats', () => {
+    typeCode('<Foo><Foo><Bar />');
```

Comment:
Should these be self closing?

Reply:
Nah, because when the browser types the closing brace `>` the closing tag is auto inserted after the cursor.

Reply:
Oh, right.